### DISCUSSION TITLE
LBB Causal Inference

### DISCUSSION THREAD

**Discussion by dimasrepo** [teamalgoritma/community#929] [Q&A] [answered]
Dear Team,
Saya sedang mengerjakan LBB Causal, ada sedikit kebingungan untuk menentukan tujuan dan Causal Graphnya, apakah graph dibawah ini sudah benar ? Kemudian apakah variable weekend dan business name ini bisa dihapus saja atau bagaimana ya ? karena dilihat tidak memiliki hubungan untuk pengolahan kausalnya 

![image](https://github.com/user-attachments/assets/9918ee84-7879-4873-99bf-9a5373a1e124)

Berikut deskripsi projectnya

# Pengaruh Jumlah Lokasi terhadap Skor Inspeksi

## Tujuan
Tujuan dari analisis ini adalah untuk menilai apakah jumlah lokasi yang dimiliki oleh restoran (`NumberofLocations`) berpengaruh terhadap skor inspeksi yang diterima oleh restoran (`inspection_score`).

## Hipotesis
- **Hipotesis Nol (H₀)**: Tidak ada pengaruh signifikan antara jumlah lokasi restoran (`NumberofLocations`) terhadap skor inspeksi yang diterima oleh restoran (`inspection_score`).
  
- **Hipotesis Alternatif (H₁)**: Ada pengaruh signifikan antara jumlah lokasi restoran terhadap skor inspeksi yang diterima. Semakin banyak lokasi yang dimiliki restoran, semakin tinggi atau lebih rendah skor inspeksi yang diterima.

## Penjelasan Hipotesis:
- **H₀** mengasumsikan bahwa jumlah lokasi restoran (`NumberofLocations`) tidak mempengaruhi skor inspeksi yang diterima restoran. Dengan kata lain, restoran dengan satu atau banyak lokasi akan memiliki skor inspeksi yang serupa.
  
- **H₁** menyatakan bahwa jumlah lokasi yang dimiliki restoran berpengaruh terhadap skor inspeksi yang diterima. Semakin banyak lokasi yang dimiliki restoran, dapat berhubungan dengan peningkatan atau penurunan skor inspeksi. Hal ini bisa terjadi karena restoran dengan lebih banyak lokasi mungkin memiliki sistem manajemen yang lebih baik atau lebih buruk dibandingkan restoran dengan sedikit lokasi.

## Variabel yang Terlibat:
- **Variabel Dependen (Y)**: `inspection_score` – skor inspeksi yang diterima oleh restoran, yang diukur pada skala tertentu (misalnya 0-100).
  
- **Variabel Independen (X)**: `NumberofLocations` – jumlah lokasi yang dimiliki oleh restoran.
  
- **Variabel Kontrol**:
  - `Year` – tahun inspeksi dilakukan, yang dapat mempengaruhi skor inspeksi.
  - `Weekend` – apakah restoran buka pada akhir pekan, yang mungkin berhubungan dengan volume pengunjung dan skor inspeksi.

## Penjelasan Analisis:
Untuk menguji hipotesis ini, kita bisa menggunakan **model regresi linier** atau **model regresi lainnya** yang memungkinkan kita untuk mengukur hubungan antara `NumberofLocations` dengan `inspection_score`, sambil mempertimbangkan variabel kontrol seperti `Year` dan `Weekend`.

Dengan menganalisis data ini, kita dapat mengetahui apakah jumlah lokasi yang dimiliki restoran berperan dalam menentukan seberapa baik atau buruk restoran dalam hal skor inspeksi yang diterima.

Best Regards 
Dimas

**Comment by hwulanayu** [comment, marked as answer]
Dear @dimasrepo,

Dalam membuat Causal Graph, tidak semua variable yang ada pada suatu dataset harus dimasukan ke dalam graph tersebut. Hal ini sangat bergantung kepada domain knowledge Bapak. Jika suatu variable diasumsikan tidak berpengaruh signifikan terhadap variable target/outcome, Bapak bisa hilangkan variabel tersebut. Terkait deskripsi projeknya, hati-hati jika **hanya** menggunakan model regresi linier biasa untuk sample data observasional. Kemungkinan besar tidak memberikan efek kausal yang tepat karena ada asumsi yang cukup sulit dipenuhi, yaitu Bapak sudah memastikan untuk memasukan semua variable confounder ke dalam model regresi tersebut. 

Best Regards,
Team Algoritma

**Reply by IrfanChairurrachman** [reply]
Dear @dimasrepo 

Terima kasih telah bertanya melalui Github discussions Algoritma. Untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami akan menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.

Best regards,
Team Algoritma
